frontends/web/src/routes/device/bitbox02/passphrase.tsx

```diff
@@ -56,77 +50,46 @@ export const Passphrase = ({ deviceID }: TProps) => {
         return;
       }
       const { mnemonicPassphraseEnabled } = result.deviceInfo;
-      // before enabling/disabling we show 1 or more pages to inform about the feature
-      // each page has a continue button that jumps to the next or finally toggles passphrase
-      // infoStep counts down in decreasing order
       setPassphraseEnabled(mnemonicPassphraseEnabled);
-      setInfoStep(mnemonicPassphraseEnabled ? INFO_STEPS_DISABLE : INFO_STEPS_ENABLE);
     }).catch(console.error);
-  }, [deviceID, setPassphraseEnabled, setInfoStep, t]);
+  }, [deviceID, t]);
 
-  const togglePassphrase = async () => {
+  const setPassphrase = async (enabled: boolean) => {
```

Comment:
just nit: it all makes sense but we have now `setPassphrase` and `setPassphraseEnabled` with very similar naming this is slightly confusing.

Not sure maybe change the state to `isEnabled`/`setIsEnabled` or something better?

Reply:
(good point, pushed)

Reply:
thanks